src/openeo_grass_gis_driver/collections.py

```diff
@@ -93,6 +94,7 @@ def get(self):
             # Additionally check for STAC collections registered in actinia
             status_code, stac_collections = self.iface.get_stac_collections()
             if status_code != 200:
+                log.warning("Couldn't get STAC collections from actinia")
                 stac_collections = []
 
             for i in stac_collections['collections']:
```

Comment:
Unrelated to this PR:
This will cause a key value error if `stac_collections = []`. Must be protected.

Reply:
:+1: will do this in a follow up